src/lib.rs

```diff
@@ -975,4 +1317,4 @@ pub struct RenameParams {
     /// appropriate message set.
     #[serde(rename="newName")]
     pub new_name: String,
-}
+}
```

Comment:
Could you add a newline to the end of the file? It makes diffs later on cleaner.